src/tree.rs

```diff
@@ -58,7 +58,12 @@ impl From<MerkleHashOrchard> for Anchor {
 }
 
 impl Anchor {
-    pub(crate) fn empty_tree() -> Anchor {
+    /// The anchor of the empty Orchard note commitment tree.
+    ///
+    /// This anchor does not correspond to any valid anchor for a spend, so it
+    /// may only be used for coinbase bundles or in circumstances where Orchard
+    /// functionality is not active.
```

Comment:
It is entirely valid to use the empty tree in regular bundles. Doing so is equivalent to setting `enable_spends = false` - it proves publicly that the Orchard bundle spends no notes (but may still have real outputs). I think this comment is fine however because callers should reach for `enable_spends` in this case, rather than the empty anchor.